test/storage/index.js

```diff
@@ -192,8 +192,4 @@ describe('Storage', function() {
       });
     });
   });
-
-  describe('makeReq_', function() {
```

Comment:
Git blame blames you @stephenplusplus . https://github.com/GoogleCloudPlatform/gcloud-node/blame/master/test/storage/index.js#L196